### DISCUSSION TITLE
Is it possible to default send-later to put into Outbox?

### DISCUSSION THREAD

**Discussion by adamlau** [Extended-Thunder/send-later#341] [Q&A]
We have a process where generated emails (from a legacy MAPI client app) need to be manually edited/checked before actually being sent. I have enabled the option for "Send does Send Later" (and disabled "Trigger unsent message delivery from outbox") and can then choose from the pop-up dialog to put in the Outbox. Is there a way that we can get the email to default to put in outbox without prompting.?

**Comment by jph-sendlater** [comment]
That's the behavior of Thunderbird's built-in send later function. You have a few options:

1. If you don't use the scheduling functionality in the Send Later extension you could just disable the add-on altogether. That way Ctrl+Shift+Enter or File>Send Later would always put the message directly in the outbox.
2. You can use the option "Bind Alt-Shift-Enter instead of Ctrl-Shift-Enter", which passes Ctrl-Shift-Enter on to the built-in function (places directly in outbox)
3. You can use the keyboard shortcut when the scheduler dialog opens (depending on your language and operating system it's something like Alt+Shift+O)
4. You can setup a shortcut for "Delay: 0 minutes", which will save the message to drafts and move it over to the outbox during its next check.

**Reply by adamlau** [reply]
Thanks for the quick reply. Turns out the legacy app wasn't going to play nice with Thunderbird anyway. This is a great extension and will keep it in mind for the future!
